introduction/java_loops_II

```diff
@@ -0,0 +1,33 @@
+import java.lang.Math;
+import java.lang.Object;
+import java.lang.StringBuilder;
+import java.util.Scanner;
+
+class Solution {
+  private static final int B_MULTIPLIER_BASE = 2;
+  
+  private static String generateSeries(int coefficientA, int coefficientB, int seriesLength) {
+    int bMultiplier = 1;
+    int seriesTerm = coefficientA;
+    StringBuilder resultBuilder = new StringBuilder();
+    for (int i = 0; i < seriesLength; i++) {
+      seriesTerm += coefficientB * bMultiplier;
+      resultBuilder.append(seriesTerm + " ");
+      bMultiplier = bMultiplier * B_MULTIPLIER_BASE;
+    }
+    return resultBuilder.toString();
+  }
+  
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
```

Comment:
Note how you use input and output variables and how ambiguous they are. You use input in a sense "This is a source of input data" and output in a sense "This is data to be written to output". Better to avoid this, as it's confusing.